### PR TITLE
Replaced stylesheets with theme provider and added a seperator to the lessons list

### PR DESCRIPTION
**Pull request by markusamuel**

### CODE REVIEW

**Review by mark-yacoub — CHANGES_REQUESTED** [review]
please watch oout for ALL warnings and errors. I see this when i run `yarn`
image

File: src/screens/level1/learningLetters.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import {View, Text, TextInput, SafeAreaView, ScrollView, StyleSheet, StatusBar} from 'react-native';
```
Comment (mark-yacoub) [inline src/screens/level1/learningLetters.js:2]: n5alni balna ya bro - seems untested:
image

Comment (markusamuel) [reply src/screens/level1/learningLetters.js:2]: good catch @mark - commit amended

**Comment by mark-yacoub** [comment]
you should not have merged if you hadn't received an approval.

**Comment by markusamuel** [comment]
best practise or something gone wrong ?

**Pull request merged**